pkg/operator/status.go

```diff
@@ -289,9 +299,172 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if updating or degraded
+	if !updating && !degraded {
+		skewStatus, status, err := optr.isKubeletSkewSupported(pools)
+		if err != nil {
+			coStatus.Reason = status.Reason
+			coStatus.Message = status.Message
+			return optr.updateStatus(co, coStatus)
```

Comment:
Can we log error message here as well so that operator pod log captures it?

Reply:
Added glof.Fatalf

Reply:
we don't want Fatalf, that will cause the controller to exit.

Reply:
fixed.